shell/platform/common/cpp/text_input_model.h

```diff
@@ -77,9 +90,12 @@ class TextInputModel {
   // Returns true if the cursor could be moved.
   bool MoveCursorToEnd();
 
-  // Get the current text
+  // Gets the current text
   std::string GetText() const;
 
+  // Gets the cursor position as a byte offset in string returned from GetText()
```

Comment:
Even though it should be clear from the types, I think it's worth explicitly calling out that this is UTF-8 here since the API has both UTF-8 and UTF-16 in it.